kanidmd/src/lib/core/https/v1.rs

```diff
@@ -0,0 +1,896 @@
+use crate::event::AuthResult;
```

Comment:
Assuming this is just moved from `https/mod.rs`.

Reply:
Indeed!